src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { NewTodo } from '../types/NewTodo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (userId: number, data: NewTodo) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, data);
+};
+
+export const patchTodo = (todoId: number, data: any) => {
```

Comment:
Avoid using 'any' as a type. Consider creating a specific type or using Partial<Todo> for patch data.